types/swagger2openapi/index.d.ts

```diff
@@ -0,0 +1,145 @@
+    externals: any[]; // Output Information required to unresolve a resolved definition back into its component parts
+    metadata: object; // Output Used by the validator, if options.text is a string, will have a property lines containing the number of lines in the input document.
+    openapi: OpenAPIV3.Document; // Output The OpenApi 3.x definition returned from a conversion step
+    operationIds: string[]; // Output Used by validation to track uniqueness of operationIds
+    original: OpenAPIV2.Document; // Bi-directional Used by testRunner to round-trip the original definition, set by non-object ConvertXXX methods
+    patches: number; // Output Count of number of patches applied during conversion
+    sourceYaml: boolean; // Output Flag set if the source string, URL or stream contained a YAML formatted definition
+    valid: boolean; // Output The result of a validation step
+    warnings: any[]; // Output Warnings generated by a validation step
+}
+
+export function convertObj(
+    schema: OpenAPIV2.Document,
+    options: Partial<ConvertInputOptions>,
+): Promise<ConvertOutputOptions>;
+export function convertObj(
+    schema: OpenAPIV2.Document,
+    options: Partial<ConvertInputOptions>,
+    callback: (err: S2OError | undefined, options: ConvertOutputOptions) => void,
+): void;
+
+export const convert: typeof convertObj;
+
+export function convertFile(filename: string, options: Partial<ConvertInputOptions>): Promise<ConvertOutputOptions>;
+export function convertFile(
+    filename: string,
+    options: Partial<ConvertInputOptions>,
+    callback: (err: S2OError | undefined, options: ConvertOutputOptions) => void,
+): void;
+
+export function convertUrl(url: string, options: Partial<ConvertInputOptions>): Promise<ConvertOutputOptions>;
+export function convertUrl(
+    url: string,
+    options: Partial<ConvertInputOptions>,
+    callback: (err: S2OError | undefined, options: ConvertOutputOptions) => void,
+): void;
+
+export function convertStr(str: string, options: Partial<ConvertInputOptions>): Promise<ConvertOutputOptions>;
+export function convertStr(
+    str: string,
+    options: Partial<ConvertInputOptions>,
+    callback: (err: S2OError | undefined, options: ConvertOutputOptions) => void,
+): void;
+
+export function convertStream(readable: Readable, options: Partial<ConvertInputOptions>): Promise<ConvertOutputOptions>;
+export function convertStream(
+    readable: Readable,
+    options: Partial<ConvertInputOptions>,
+    callback: (err: S2OError | undefined, options: ConvertOutputOptions) => void,
+): void;
+export const targetVersion: string;
+
+declare const converter: {
+    S2OError: typeof S2OError;
+    targetVersion: typeof targetVersion;
+    convert: typeof convert;
+    convertObj: typeof convertObj;
+    convertUrl: typeof convertUrl;
+    convertStr: typeof convertStr;
+    convertFile: typeof convertFile;
+    convertStream: typeof convertStream;
+};
+
+export default converter;
```

Comment:
this one can be removed, you've already export symbols from the module (directly, not via single symbol). Direct exports reflects native module exports already:
https://github.com/Mermade/oas-kit/blob/main/packages/swagger2openapi/index.js#L1658-L1665
this will remove error you're facing and a need for `export default`

Reply:
No, the following imports will not work without export default

```ts
import converter  from 'swagger2openapi';

console.log(converter.convert)
```